docs/changelog.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-### 08-April-2021 - 17:01 CET
+### 30-April-2021 - 13:52 CEST
+
+- [feature] Add mark in logs to indicate output has been trimmed.
+- [feature] Stop uploading packages to Bintray.
+- [job] RequestReviews: Automatically request team reviews for PRs ready for review.
```

Comment:
I am super curious, how did you guys define "ready for review"?

Reply:
same as described in [docs](https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#automatic-merges) minus reviews, so effectively:
- Author is an added beta user.
- Author has signed CLA.
- PR is not a Draft or WIP.
- PR has a green status (successful build).
- PR doesn't have merge conflicts with master branch.

Reply:
Thanks!